src/website/src/app/documentation/demos/a11y/accessibility.demo.html

```diff
@@ -0,0 +1,67 @@
+<clr-doc-wrapper [ng]="ng" [ui]="ui" [title]="title" [newLayout]="newLayout">
+  <article>
+
+    <div id="guidelines">
+      <h3 id="accessibility">Accessibility</h3>
+      <p>
+        Although Clarity tries to cover as much as possible all best practices for being a аccessibility compliant. Some cases
+        are too domain-specific so to get the best experience there some tools below that are designed to help you.
+      </p>
+
+      <h3 id="aria-live-region">Aria live region</h3>
+      <p>
+        The possibility of having a dynamic component that is changing the content, notifications or loading states could create
```

Comment:
Rewrite the content of this page to have just the following.

----

When content changed dynamically on the page outside of the current focus, a screen reader user would not be aware of those changes. In these scenarios, the browser supports the `aria-live` attribute to make important announcements. You might announce errors in a form, status updates for long running tasks, or alert messages.

There are limitations to how different screen readers and assistive technologies support `aria-live`, and we've provided a service `ClrAriaLiveService` for your Angular application to use anytime you need to make an announcement in a fully compatible way.

Clarity does not make any announcements by default, it is up to the application to do this when it makes sense. Here are a few examples of how the service works and could be used.

SAMPLE 1

Aria live regions have three levels, `off` to disable it, `polite` to announce to the user at the next opportunity (default), and `assertive` to alert the user immediately and interrupt any other announcements in progress.

SAMPLE 2

A working example of using `ClrAriaLiveService` with a download, asynchronous process is below. You start by injecting the service, and based on changes in the progress state it will announce updates of the percentage updates. You might also want to debounce messages if you expect to get them very quickly so you don't have too many rapid announcements and annoy the users.

SAMPLE 3